toolkit/scss/shared_scss/_dmspeak.scss

```diff
@@ -20,6 +20,10 @@
     @extend %heading-small;
   }
 
+  .heading-large {
```

Comment:
I'd be tempted to order these small, medium, large in the file.  Or large, medium small.  But not medium, small, large.

Reply:
Good spot - now fixed.